### ISSUE TITLE
Tweaks to the Spot newsletter

### ISSUE THREAD

**Issue by githamm** [denverpost/rup#1]
Style changes:

* [x] Change font to Georgia, serif
* [x] Make main headline available: h4 (maybe) font-size:2em; line-height:115%; margin-bottom:.17em;
* [x] h2 tags --font-size:.95em; color:#3e009f;

Structure/functionality changes:
* [x] Reorder lead content chunk: Intro first, main image, caption, main headline
* [x] Lose byline (or make optional)
* [x] Create ability to add new "section" comprised of: hr h4
* [ ] Add "forward to a friend / subscribe" icons and functionality (see note)

Note: The forward to a friend functionality opens a new form so might be a feature for down the road? WSJ example: https://capitaljournal.cmail20.com/t/ViewEmail/d/B3B27EBC728219FE2540EF23F30FEDED/35EC4D1BB8C61DAEF99AA49ED5AF8B9E

Background: https://docs.marketo.com/display/public/DOCS/%27Forward+to+a+Friend+Link%27+in+Emails

**Comment by freejoe76** [comment]
@githamm question: in this issue you say you want the h2 font size to be .95 em... that feels small. The photo caption font size is .85 em. This is what .95 looks like in a browser:

![screen shot 2018-08-30 at 3 41 49 pm](https://user-images.githubusercontent.com/125554/44877966-3d9d5780-ac6b-11e8-97d1-7cad7c66afb4.png)

**Comment by freejoe76** [comment]
Also, in the issue, what do you mean by "main" when you write "Make main headline"?

**Comment by freejoe76** [comment]
Also, re: reordering the main item in the newsletter, I believe that's something that would be done by the author in WordPress, no?

**Comment by githamm** [comment]
- h2 main hed should be default font color
- h2 non-main hed should be the purple #3e009f and have a rule above it -- that's the "module" they want to be able to drop in to indicate a new topic ... the only two heading types would be the h2 main and h2 non-main, for lack of a better term

**Comment by githamm** [comment]
Markup and icon file for "forward to a friend" widget ...

`<p style="border-top:1px solid silver;border-bottom:1px solid silver;padding:1em;text-align:center;">
                                        <img style="width: 3em;display:block;margin:0 auto;padding-bottom:.5em;" src="email-icon-modified.png">
                                        <span style="font-family:sans-serif;font-size:.95em;color:#000;font-style:italic;">Did a friend forward the Spot newsletter to you? <a href="https://www.denverpost.com/newsletter/" style="color:#CE4815;text-decoration:none;">Subscribe here</a>.</span>
                                    </p>`

![email-icon-modified](https://user-images.githubusercontent.com/6498109/44880388-8a802e80-ac69-11e8-9820-3b26b2c11e46.png)

Looks like (on mobile) ...

![subscribe](https://user-images.githubusercontent.com/6498109/44880410-97048700-ac69-11e8-8104-0d45111d4f5d.png)

Order of newsletter goes: intro > main story > share > sections...

And we need to credit the email icon somewhere near the bottom: Created by H Alberto Gongora from Noun Project

**Comment by freejoe76** [comment]
Okay great, that helps with my clarity about the h2's. I'm not sure the share part will be finished by Tuesday. Will there be a blurb of text under the h2 main? There will be, won't there.

**Comment by githamm** [comment]
Yep. It's my understanding that it will be something like (and I'm calling the purple h2s "subhed" to differentiate it from the main hed (there will be only one main hed in the newsletter):

logo > tagline > intro > image > cutline > main hed > body text > share widget > subhed module > subhed module > subhed module > subhed module (however many they want, with images probably sprinkled throughout)
